README.md

```diff
@@ -79,6 +94,34 @@ artefactor restore --source-dir ~/
 `artefactor publish` takes files from the relative ./downloads path and 
 publishes containers / files to any remote registries / locations.
 
+### update-image-vars
+
+Artefactor can update environment variables with a list of transformed image
+names. This is useful when environment variables with image names are used in
+deployments and they need to be managed with a private registry name.
+
+Given the following shell variables:
+```bash
+MYSQL_IMAGE=mysql:v5.0
+CASSANDRA_IMAGE=docker.io/cassandra:latest
+ARTEFACTOR_IMAGE_VARS="MYSQL_IMAGE CASSANDRA_IMAGE"
+ARTEFACTOR_PRIVATE_REGISTRY=myreg.local
+```
+
+To update environment variables with a list of images (as discovered from the
+ downloaded artefacts):
+`eval $(artefactor update-image-vars)`.
```

Comment:
May need to be a little careful here in case the command produces unexpected output such as errors etc.